.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+name: release
```

Comment:
What's the purpose of this workflow? To just showcase an example of a workflow that uses the new action? If so, I'd not call this workflow "release" since it does not seem to be releasing anything, and in fact we are going to soon add a release workflow that actually builds, tags and publishes a Github Release.